RevenueCatUI/Data/LocalizedAlertError.swift

```diff
@@ -14,28 +14,25 @@
 import RevenueCat
 import SwiftUI
 
-struct LocalizedAlertError: LocalizedError {
-
-    private let underlyingError: NSError
```

Comment:
I would keep the implementation with `underlyingError` as is. Is there any particular reason to store all the values in separate variables?

Reply:
Resolving this, as we need a way to pass these values directly